source/declarations.tex

```diff
@@ -227,12 +227,12 @@
 
 \pnum
 A \grammarterm{nodeclspec-function-declaration} shall declare a
-constructor, destructor, or conversion function.  \footnote{The
+constructor, destructor, or conversion function.\footnote{The
 ``implicit int'' rule of C is no longer supported.}
 \begin{note}
 A \grammarterm{nodeclspec-function-declaration} can only be used in a
 \grammarterm{template-declaration}~(Clause~\ref{temp}),
-\grammarterm{explicit-instantiation}~(\ref{temp.explicit}), or
+\grammarterm{explicit-instan\-tiation}~(\ref{temp.explicit}), or
```

Comment:
Done, although I notice that we do usually attach cross-references to the preceding term.